internal/coremsgs/en_error_messages.go

```diff
@@ -256,4 +256,5 @@ var (
 	MsgActionNotSupported                 = ffe("FF10414", "This action is not supported in this namespace", 400)
 	MsgMessagesNotSupported               = ffe("FF10415", "Messages are not supported in this namespace", 400)
 	MsgInvalidSubscriptionForNetwork      = ffe("FF10416", "Subscription name '%s' is invalid according to multiparty network rules in effect (network version=%d)")
+	MsgBlockchainNotConfigured            = ffe("FF10417", "No blockchain plugin configured", 400)
```

Comment:
nit: not sure this is an HTTP 400, as it's not explicitly a problem with the request format (more of a server config error, so probably can omit the error code and keep the default of 500)